composite-base/src/Composite/Record.hs

```diff
@@ -227,6 +233,33 @@ rlens' proxy f =
     fmap Val <$> f fa
 {-# INLINE rlens' #-}
 
+-- |Lens to a particular field of a record using a contravariant functor.
+--
+-- For example, given:
+--
+-- @
+--   type FFoo = "foo" :-> Int
+--   type FBar = "bar" :-> String
+--   fBar_ :: Proxy FBar
+--   fBar_ = Proxy
+--
+--   rec :: 'Rec' 'Predicate' '[FFoo, FBar]
+--   rec = Predicate even :!: Predicate (even . length) :!: Nil
+-- @
+--
+-- Then:
+--
+-- @
+--   view (rlens'' fBar_)                           rec == Predicate even
+--   set  (rlens'' fBar_) Predicate (odd . length)  rec == Predicate even :!: Predicate (odd . length) :!: Nil
+--   over (rlens'' fBar_) (contramap show)          rec == Predicate even :!: Predicate (odd . length . show) :!: Nil
+-- @
+rlens'' :: (Contravariant f, Functor g, RElem (s :-> a) rs) => proxy (s :-> a) -> (f a -> g (f a)) -> Rec f rs -> g (Rec f rs)
```

Comment:
I think it would be nice to have a better name for this - `rlens'` is already a confusing enough name for what it does. maybe `rlensContra` and then rename (with backwards compatibility alias) `rlens'` to `rlensCo`?